sdk/keyvault/keyvault-keys/src/core/keyVaultClient.ts

```diff
@@ -21,10 +21,7 @@ class KeyVaultClient extends KeyVaultClientContext {
    * @param credentials Credentials needed for the client to connect to Azure.
    * @param [options] The parameter options
    */
-  constructor(
-    credentials: coreHttp.TokenCredential,
-    options?: coreArm.AzureServiceClientOptions
-  ) {
+  constructor(credentials: coreHttp.TokenCredential, options?: coreArm.AzureServiceClientOptions) {
```

Comment:
We need to speak with @daviwil and @schaabs to make sure that we can apply these changes to a future version of our core files, since they're auto-generated.